tests/plugins/test-woocommerce-memberships.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Class Newsletters Test Woocommerce Memberships.
+ *
+ * @package Newspack_Newsletters
+ */
+
+namespace Newspack_Newsletters\Plugins;
+
+/**
+ * Test Woocommerce Memeberships.
+ */
+class Woocommerce_Memberships_Test extends \WP_UnitTestCase {
+	public function test_is_membership_list() { // phpcs:ignore Squiz.Commenting.FunctionComment.Missing
+		$this->assertTrue( Woocommerce_Memberships::is_membership_list( 1 ) );
+		$this->assertFalse( Woocommerce_Memberships::is_membership_list( 0 ) );
```

Comment:
These tests don't make the use of the method clear. After digging into the code, I see that `0` is an id of a non-existing list, so the method returns false. `1` is apparently an id of a list tied to a membership plan, but it's not evident anywhere in the test. 
A test should be able to be read as a component spec – _given these conditions, code does that, etc_. In this test, the subscription list tied to a membership plan should first be set up, then it's id used in the method test.

Reply:
I see what you mean. I had a hard time moving the setup of the membership plan into the test since getting the memberships relies on the global `wc_memberships_get_membership_plans` method and as far as I can tell, there isn't a simple way to mock this from within the test without adding some more complexity to the wc-memberships mock.

That said, I did move things around and move the setup into a static helper method to try and make things more clear in https://github.com/Automattic/newspack-newsletters/pull/1502/commits/33fae942c8c09f32f60df78e6a49b4976bebd96e

Let me know if this is good enough or if you think there is another strategy I should take here.

Reply:
Much better!